IDEOorg/steps
# #161240638 Write Tests for SortableTaskList.tsx

#### What does this PR do?
Add tests for SortableTaskList.tsx
#### Description of Task to be completed?
- Add tests for SortableTaskList.tsx
#### How should this be manually tested?
- Clone the repository from `https://github.com/IDEOorg/steps.git` and navigate into the folder `cd steps`
- Check out the branch `git checkout ch-161240638-write-sortable-task-list-tests`
- Follow the instruction here `https://github.com/IDEOorg/steps#refresh-with-latest-build` to install the dependencies
- Navigate into the coach app `cd app/admin`
- Run the tests `npm test`
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#161240638](https://www.pivotaltracker.com/story/show/161240638)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
